include/libhal-input/ch9329.hpp

```diff
@@ -267,6 +342,13 @@ class ch9329
    * @param p_data mouse relative object containing command bytes
    */
   void send(mouse_relative const& p_data);
+  /**
+   * @brief Send keyboard media command
+   *
+   * @param p_data keyboard media object containing command bytes
+   * @param p_is_acpi is the command sending acpi key presses
+   */
+  void send(keyboard_media const& p_data, bool p_is_acpi = false);
```

Comment:
Is there a way to infer this based on the keyboard_media data? Because what happens if I forget to set this to true?